src/EnergyPlus/SizingManager.cc

```diff
@@ -709,6 +710,16 @@ void ManageSizing(EnergyPlusData &state)
         state.dataSize->SysSizing.deallocate();
     }
 
+    // Size SpaceHVAC fractions
+    if (state.dataHeatBal->doSpaceHeatBalanceSimulation) {
+        for (auto &thisSpaceHVACSplitter : state.dataZoneEquip->zoneEquipSplitter) {
+            thisSpaceHVACSplitter.size(state);
+        }
+        for (auto &thisSpaceHVACMixer : state.dataZoneEquip->zoneEquipMixer) {
+            thisSpaceHVACMixer.size(state);
+        }
+    }
+
```

Comment:
A little out of the ordinary where sizing occurs on the call to each object, but OK, this happens after system sizing so space splitter/mixer can be sized.

Reply:
> A little out of the ordinary where sizing occurs on the call to each object, but OK, this happens after system sizing so space splitter/mixer can be sized.

Yah, fer sure dat. This was a bit of a punt. Would love a better suggestion from the sizing guru, for another day.